apps/system/utils/kdbg_date.c

```diff
@@ -67,6 +67,9 @@
 #define MAX_TIME_STRING         80
 #define STR_LEN_OF_MONTH_WEEK   3
 #define MAX_EPOCH_YEAR_OF_UINT  2106
+#define EPOCH_YEAR_CONST        70
```

Comment:
This is not fixed. This value is depends on TM_YEAR_BASE and EPOCH_YEAR.
Let's modify this like below
```
#define EPOCH_YEAR_CONST   (TM_YEAR_BASE - EPOCH_YEAR)

Reply:
`TM_YEAR_BASE -> 1900` and `EPOCH_YEAR -> 1970`, So the definition should be ```#define EPOCH_YEAR_CONST   (EPOCH_YEAR - TM_YEAR_BASE )```

@sunghan-chang find the expected result below,
```
TASH>>
TASH>>date -s "Jan 01 12:00:00 1969"
date: cannot set date: Invalid year in the argument!

TASH>>
TASH>>
TASH>>date -s "Jan 32 12:00:00 2018"
date: cannot set date: Invalid date-time argument!

TASH>>date -s "Jan 01 12:00:00 1923"
date: cannot set date: Invalid year in the argument!

TASH>>
```